web/src/components/AnalyzeData.vue

```diff
@@ -0,0 +1,35 @@
+<script>
+import Vue from 'vue';
+import { analyses } from './vis';
+
+export default Vue.extend({
+  props: {
+    id: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      analyses,
```

Comment:
Is there a reason this is not passed in as a prop?

Reply:
This component is called from the router, so there is nothing to fulfill that prop.

Reply:
Can't we pass that prop in using ["object mode"](https://router.vuejs.org/guide/essentials/passing-props.html#object-mode)?

Reply:
No, we still need id from router mode.